src/plugins/base-formatters.js

```diff
@@ -0,0 +1,139 @@
+
+function plugin({ Formatter, formatterRegistry }) {
+    Formatter.colors.fromLevel = (severity) => {
+        let severityString = severity.name;
+        switch (severityString) {
+            case 'INFO':
+                return Formatter.colors.blue(severityString);
+            case 'DEBUG':
+                return Formatter.colors.brightBlue(severityString);
+            case 'NOTICE':
+                return Formatter.colors.cyan(severityString);
+            case 'WARNING':
+                return Formatter.colors.brightYellow(severityString);
+            case 'ALERT':
+            case 'CRITICAL':
+                return Formatter.colors.yellow(severityString);
+            case 'ERROR':
+                return Formatter.colors.red(severityString);
+            default:
+                return severityString;
+        }
+    }
+
+    formatterRegistry.add('JSON', {
+        formatLog: (log) => JSON.stringify(log),
+        color: (log) => log,
+    });
+
+    formatterRegistry
+        .register(
+            'MESSAGE',
+            '{message}',
+        )
+        .register(
+            'SHORT',
+            '[{time}] - {level} - {message}', {
+            props: {
+                time: {
+                    transformers: ['toLocaleDate'],
+                },
+                level: {
+                    transformers: ['toString']
+                },
+                message: {
+
+                }
+            }
+        }
+        )
+        .register(
+            'MEDIUM',
+            '[{time}] - {level} - {message} {data}',
+            {
+                props: {
+                    time: {
+                        transformers: ['toLocaleDate']
+                    },
+                    level: {
+                        transformers: ['toString']
+                    },
+                    message: {},
+                    data: {
+                        transformers: ['stringify', 'cl_gray']
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/loggin-js/loggin-js/pullRequest?prid=7629800)
```suggestion
                        transformers: ["stringify", 'cl_gray']
```